platform/ui/src/components/StudyListFilter/StudyListFilter.tsx

```diff
@@ -69,16 +69,16 @@ const StudyListFilter = ({
                 )}
                 <Typography
                   variant="h6"
-                  className="mr-2"
-                  data-cy={'num-studies'}
+                  className="text-primary-light"
```

Comment:
I kind of fixed that in the orth i18n PR since it was failing CI so you can revert this